packages/flutter_tools/test/general.shard/web/asset_server_test.dart

```diff
@@ -78,4 +78,18 @@ void main() {
 
     expect(response.statusCode, 404);
   }));
+
+  test('release asset server serves correct mime type and content length', () => testbed.run(() async {
```

Comment:
Few comments about the tests:
- Does it make sense to have a test that exercises the inference based on `headerBytes`? (i.e. omit the file extension).
- Can we add tests for the most important mime types (`text/javascript` and `text/html`)?

Reply:
Updated to add more test cases

Reply:
Thanks!